### DISCUSSION TITLE
More gender neutral language in German translations

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#28221] [General]
Currently, every user is called 'ein Benutzer' and the phrasing shows a clear assumption of a male user. Example:
"______ ist ein Benutzer auf witches.town. Du kannst ihm folgen oder mit ihm interagieren."

This thread below explains the need for more gender neutral language in the German translation of Mastodon, as well as some limitations and ideas from an accessibility and trans-inclusion perspective.
`https://discourse.joinmastodon.org/t/de-making-the-german-translation-more-gender-neutral-while-also-accessible/468`

My thoughts:

- Considering that people are not inputting their gender to Mastodon, having one gender neutral declination for all users seems like the best option.

- Because not every user is an 'er' or 'sie', the form used should be inclusive of non-binary people. (So, not 'er/sie')

- Ideally, this should be accessible to screen readers. It would be best to avoid special characters.

I have two proposals!

1. I propose to use either the ending -er_in for the suffix, and to use 'ein_e' for the article. For example 'ein_e Benutzer_in'. When it comes to the pronoun, I propose to use 'xier'. This option is admittedly not screen reader friendly but it's the most common way of referencing people in a gender neutral way in places that I am familiar with

    "______ ist ein_e Benutzer_in auf witches.town. Du kannst xiem folgen oder mit xiem interagieren."

2. The second proposal is to use the ending -lon, and to use the declination of 'ein' which corresponds with the neuter grammatical gender. For example 'ein Benutzlon'. Again I propose to use 'xier' as the pronoun, or another gender neutral pronoun if people are interested. 

    "______ ist ein Benutzlon auf witches.town. Du kannst xiem folgen oder mit xiem interagieren."

I hope that this issue is resolved. From many people I hear that changing just to a feminine standard and not masculine would be a huge change. But since gender neutral language is so important, and so rare in Germany, it would be great to set high goals and for Mastodon to be a place with progressive and inclusive use of language.

Finally, I apologize if the way I'm talking about German grammar is hard to understand or incorrect, it's not my first language!

* * * *

- [x ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by averageunicorn** [comment]
All of this! I'm the person who started the discourse thread and then got distracted, and I'm super happy that someone is picking this up.

I like both proposals, but I prefer the second one with -lon.

Xier is great imo. I'd be equally happy with other neopronouns, [nin, lon or hen](http://de.nichtbinaer.wikia.com/wiki/Pronomen), for example:
"______ ist ein Benutzlon auf witches.town. Du kannst nim folgen oder mit nim interagieren."
"______ ist ein Benutzlon auf witches.town. Du kannst lon folgen oder mit lon interagieren."
"______ ist ein Benutzlon auf witches.town. Du kannst ham folgen oder mit ham interagieren."

This is not meant to distract from the big picture. There are many great options, and replacing the "he" with _anything_ gender neutral should be a priority over discussion of the ideal solution.


[edited to add] other places where the language needs to be changed to neutral:

- Gettings Started:
– "Stummgeschaltete Benutzer" and "Blockierte Benutzer" – Benutzlons or Benutzer_innen
- /settings/profile:
– "Erlaubt dir, Nutzer zu überprüfen, bevor sie dir folgen können" – Nutzlons or Nutzer_innen
- /settings/preferences:
–"Benachrichtigungen von Nicht-Folgern blockieren" – I'd suggest "Nicht-Folgenden" here, alternatively, Nicht-Folglons or Nicht-Folger_innen
– "Benachrichtigungen von Nutzern blockieren, denen ich nicht folge" Nutzlons or Nutzer_innen
- /oauth/authorized_applications:
– "Nutzer folgen, blocken, entblocken und entfolgen" – not sure if this is singualr or a typo, so either Nutzlon/Nutzer_in, or Nutzlons/ Nutzer_innen
- /about/more:
–  "Heimat für X Benutzer" – Benutzlons or Benutzer_innen

**Comment by sistason** [comment]
My thoughts:
If nin/hen/lon does not find reception on Mastodon, maybe a rephrasing would be the common ground?

"______ besitzt einen Profil auf witches.town. Du kannst diesem folgen oder mit diesem interagieren, wenn du selbst einen Profil auf ... besitzt."

Sounds masculine, but isn't really... I would also prefer to just use lon and/or *innen for plural.


In general, I would suggest replacing "Nutzer" with "Profil", as you for example only silence accounts, not their users (sadly :>)
- "Stummgeschaltete Profile" and "Blockierte Profile"
- "Erlaubt dir, Profile zu überprüfen, bevor sie dir folgen können"
- "Benachrichtigungen von Profilen blockieren, denen ich nicht folge"
- "Profilen folgen, blocken, entblocken und entfolgen"

I don't have any suggestions for "Non-followers" and the "Home of X users", which are meant as more personal and therefore probably shouldn't dehumanized as "profiles/accounts"...
Or is there a translation for "toot" already in German? "Heimat für X _Tooters_" would maybe work.

**Comment by averageunicorn** [comment]
I do like the rephrased sentences, but I generally think that it is better to be as direct as possible. So I wouldn't be sad at all if these suggestions were chosen, but they aren't first preference either.

Maybe a combination of both would be possible. I agree that "Home for X profiles" sounds very impersonal, so Benutzlons or Benutzer_innen is better suited there, and other places could be done either way, as long as we manage to avoid naming inconsistencies that might confuse users.

> Or is there a translation for "toot" already in German? "Heimat für X Tooters" would maybe work.

"Tröt" is used in the German translation. That would be "Trötende", "Tröter_innen" or "Trötlons". It might be confusing because that is not a very straightforward way to say it, but maybe it's not, I can't really tell.

**Comment by saper** [comment]
https://discourse.joinmastodon.org/t/de-making-the-german-translation-more-gender-neutral-while-also-accessible/468/5 <-- discourse thread 

I have added some nuances there. Avoiding gendered assumptions in general would be better, trying to squeeze two forms (Benutzer_in) into one is not necessarily the best.

MediaWiki solved this problem partially by letting the user to define their own gender and the pronouns are changed accordingly (translators can use a special keywords to indicate a gendered part with `{{GENDER:...}}` tag.

**Comment by PaulFree14** [comment]
thx for bringing this up! :)

I'm for gender neutral language in any case no specific gender is ment. 
In any other case I don't understand being with language so focused on gender. 
There could be so many different pronouns describing different characters....or what ever, but somehow we decided to take the gender roles. 

What I have been seen used beside 'los' was 'is'
So like 'Menschis' 'Benutzis'. 

...there is a book called 'Suzidalien' writen in gender neutral language.

**Comment by averageunicorn** [comment]
> MediaWiki solved this problem partially by letting the user to define their own gender and the pronouns

I think that letting users define their own pronouns is a bit too complicated in the case of Mastodon, as 3rd person singular pronouns are only used in that one sentence on profile pages. Using something that is neither masculine nor feminine is the easier fix.

But if someone wants to take on the effort and include personalised profile pronouns, that'd be great of course!

> I'm for gender neutral language in any case no specific gender is ment.

That is the case in the translation of Mastodon, as Mastodon does not know the gender and/or the pronoun preferences of its users :)

> So like 'Menschis' 'Benutzis'.

I think that's great as well! It is often seen as infantilising, 
but Mastodon is a great project that has the potential to normalise neutral forms like this.

**Comment by ghost** [comment]
I don't know any German, but I support neutral language and trust those of you who are experts in the language to pick the right word(s). 

I do have a big concern about screen readers, but I hope that as long as it's a consistent change across the site, that people will be able to work out the context clues. And again, I trust you language experts to pick something that, even if it's new or different, will make sense with the context clues.

**Comment by mdik** [comment]
I think working around gendered words where possible is best (as proposed by @saper), but I'd also like generisches Femininum or Benutzer:in.

**Comment by PaulFree14** [comment]
@mdik generisches Femininum would not be the best choice Imho. If you follow the discourse outside the feminist and/or anarchist circles it tends to getting into a direction viewing feminism as something to maintain structure and culture where females will dominate male. 
...this narrative is also part of a recruitment strategy of right wing. 
Thought that's something important to keep in mind. Changing language to feminin will support it, and so their recruitment strategy. 

Sayn that, feminists of cource shouldn't let them get limited to selfempower cause of fascists, but to consider a neutral language/covering all gender instead.

**Comment by mdik** [comment]
What?

**Comment by saper** [comment]
#4789 did a pretty good job, thanks @andydrop !

```ein Benutzer``` got replaced mostly by ```ein Profil```, not too bad!

**Comment by ghost** [comment]
The credits for this idea should go to @sistason . I just decided to give them more than a like

**Comment by sistason** [comment]
Thanks and thanks to you for doing the ctrl+R-Work and thanks for @averageunicorn and @eribloodlust to start/re-start this thing! :)

So, let's recap what's left to do, since Profil only works in certain locations and there are still some harder cases to discuss, I think, namely:
- Nicht-Folgern
- Heimat von X Profilen

Nicht-Folgern is still in that form, the Heimat was changed to Profile, which is a good start, but possibly not the best solution, since we want a more humane, warm and cuddly term for our >Home< than just profiles :)

- X Menschen haben hier ihr Zuhause
- Heimat von X Trötern  (childish?)
- Heimat von X Individuen  (too artsy?^^)
- Heimat von X Nutzer*innen / Nutzer_innen / NutzerInnen / >Be<nutz...
- Heimat von X Nutzlons/Benutzlons
- ...

**Comment by ghost** [comment]
While I personally like "Tröter", a agree, that it is too childish. I would also prefer "Menschen" where possible, for the warmth that comes with it. Same goes for "Zuhause" so how about:

- X Menschen haben hier ihr Zuhause
- Diese Instanz wird von X Menschen genutzt (passiv)
- X Menschen nutzen diese Instanz (als ihr Zuhause) (aktiv)
- FÜr X Menschen ist dies ihre Heimat im Fediversum (emphasize, that there are more instances)

The obvious way for "Nicht Folgern" would be "Nicht-Folgende". But that sounds i little too non-intuitive to me.

I would like to give it some thought. Do you know the context, in which this phrase is used?

**Comment by sistason** [comment]
- "Benachrichtigungen von Nicht-Folgern blockieren" - so 'von Nicht-Folgenden' would work...
- I like your suggestions for the Heimat, I like the last one best.

But let's not get ahead by too much, we got far and I would like to use this point to invite criticism and the possibility to rethink this all / change some details.
I don't like to finish / call "well finished" by a few cis-guys what was started by the ones actually affected by the generic-masculinum-exclusion. 
So let's open this again from all angles and if we don't find a better result or other suggestions, we got a good-enough one. 
Let's start the Refactoring ;)

I start by voicing that we loose the ability to advance genderneutral pronouns, since we could integrate them in a popular platform here, to enhance visibility and make them more common :)

**Comment by ghost** [comment]
Ah, found the context in the settings. There are two settings translated:

Benachrichtigungen von Nicht-Folgern blockieren
Benachrichtigungen von Nutzern blockieren, denen ich nicht folge

The latter will be translated by the last patch to "Profile"

So how about:

Benachrichtigungen von Profilen blockieren, die mir nicht folgen.
Benachrichtigungen von Profilen blockieren, denen ich nicht folge.

**Comment by ghost** [comment]
When there are mostly positive reactions to the last proposal i will do that patch sometime tomorrow, and call that an end of my contribution.

I would not be offended, if someone (preferebly some non cis, or otherwise involved person) someday shreds my  patches by another translation.

My point of "just doing it" was like this:

- Taking away the discussion point of "People may feel misgendered", which I absolutely share.
I think when noone is gendered now, noone is misgendered.

- It's, as you stated, a "good enough" solution
 My back is wide enough to take the critics for not delivering perfection^tm

There are several reasons why one wants to use some kind of gender neutral translations, that stand out as being willingly gender neutral.
As I support some forms of it (namely Binnen-I, alternating genders, generic femininum or the gender star)
it is more of a political decision to use it in a prominent software. 

Political decisions may tear apart usergroups. That is no poblem if it is about some tool to format your hard drive, but needs some consideration if your software wants to establish a social network.

My advice would be "Choose your battles wisely"

That said i will drop my patch tomorrow, and leave the rest to the people it concerns more.

**Comment by averageunicorn** [comment]
I think that the current avoidance of talking about people is a good start. It does the job of getting rid of the "generic" masculine, and that is the most important part.

But, for the sake of feedback from one genderqueer trans person, I'm going to make this more verbose.

I have to admit that it doesn't feel great to see "weird" forms avoided at all costs, and I'll try to explain.

Imo what makes this situation a bit complicated in the first place is that -er_in and -er*in are unfortunately the most popular ungendered forms, they are even starting to spread outside of queer circles. Which would make them an obvious choice, except that they can not guarantee screen reader accessibility and are thus unsuitable for any larger scale project.

And all the other solutions (that do not avoid talking about people) either have some different kind of flaw, or take some courage to stand by (or both).

**Binnen-I** is not terrible. But unless you accompany each use with "and that includes nonbinary people!" (which, actually, is not a bad idea, and might lead to Binnen-I being a truly neutral choice over time, but "BenutzerInnen, natürlich inklusive nichtbinärer Leute!" is definitely too long for the getting started column), it is usually perceived as a very binary choice by the people who are not.

**Generic feminine** is not terrible either. But (partly because it is not as normalised as generic masculine by far), many trans people who are not women have a somewhat anxiety-loaded relationship to it. It has both the large feminist potential set the very clearly non-masculine as default, and the large un-feminist potential to create unnecessary stress for those trans people for which it has additional implications.

**Avoiding nouns that gender people**, to me, feels better than Binnen-I or Generic Feminine. At the same time, distancing myself from things that might look queer is not a business that I'm in. And I don't love the way "Profil" sounds a bit impersonal and like I'm reading a manual.

Both rephrased sentences and **-lon endings** are kind of awkward, there's no way to sugarcoat that. I prefer -lon, because it gives me a hint that the possibility of my existence was considered, because while it is not gender specific, it does originate from nonbinary trans communities.

But because both can become tiring when overused, a combination seems very reasonable to me. As in, not going out of our way (or venturing into too impersonal/abstract territory) when rephrasing, and switching to -lon instead.

In the case of **pronouns in that profile sentence**, I would very much prefer xier, nin or some other neopronoun. Both a neopronoun and the avoidance of pronouns can be confusing there, and as sistason said, using neopronouns has the additional advantage of normalising them.

We're in geek space anyway, and "different than other social media" is definitely a reason why people join and stay, so -lon and xier/nin might even be taken as lovable characteristics of Mastodon language.

No matter what solution is chosen gender-wise, I super love andydrop's suggestion to replace Heimat by Zuhause, as Heimat does have some awkwardly patriotic connotations.

**Comment by ghost** [comment]
What a massive amount of input to think about. Thank you very much @averageunicorn 

Before getting to that non-avoiding thing, some short suggestions for my gender avoiding patch: I will rephrase to:

Benachrichtigungen von {Profilen} blockieren, die mir nicht folgen.
Benachrichtigungen von {Profilen} blockieren, denen ich nicht folge.

Diese Instanz nutzen X Menschen als ihr Zuhause im Fediversum

While i like "Profile" because it does its job as being gender neutral, "Personen" oder even "Menschen" would  work the same while being less and less dehumanizing. I would like to change it to Menschen, but sometimes that sounds awkward. 

Now to the possible future translation:

If we want to do a broader approach to genders than just ignoring them (and i think we want) we seem to agree, that Binnen-I and generic feminine are not the ways to go, because they only address the corner cases of the gender spectrum.

I never heard of that -lon ending before this dicussion. Did i get it right that it is a widely agreed ending in the genderqueer community? If that is the case, it would be definitely worth going for it.

To get there i see some possible paths, which could be taken independently of each other:

1. We establish a word (in our daily toots) to adress other Mastodon users, that already ends in -lon. Over at Twitter i used to call other people Tweeties or Tweethearts (independant of their gender). Unfortunatly Mastodon is a rather poor word when it comes to wordplay. so we need some creative ideas.

2. We provide a fork, that only differs from the upstream mastodon in the german translation, for any admin interested in it.

3. We address Eugen if he would like Mastodon to be openly genderqueer supporting in the german translation. It's his poject, so he should decide if he wants to handle the flames he will likely get for that.

4. Maybe, if he doesn't want Nutzerlon to be the official german translation he has to defend, it could be possible to do an additional german language file <de-neutral> or something like that.

**Comment by averageunicorn** [comment]
> Did i get it right that it is a widely agreed ending in the genderqueer community?

Sadly, it is not. The only solutions well known across (and, in that case, beyond) genderqueer communities are * and _, and to a lesser degree x, which has some other problems. As far as I am aware, lon originated in genderqueer contexts on twitter about 2 years ago and has only spread to some other offline and online communities.

> We establish a word (in our daily toots) to adress other Mastodon users, that already ends in -lon.

I could imagine Tootlon or Trötlon. Sadly, I don't toot in German much, but I'll certainly try to use it when I do.

> Nutzerlon

Nutzlon :) lon is added before the masculine ending (or anything that looks like it), or if not possible, the lon-noun is derived from a verb. Verkäufer_in becomes Verkäuflon or Verkauflon, Pilot_in might become Pilotierlon, and Autor_in would call for more creative solutions (Schriftstelllon, Schreiblon).

I don't have an opinion on how to proceed. I certainly support additional language files. If it should come to this, I would propose (in addition to the current Profil version) one with -lon and one with -er_in, to have both the most practical and the most popular version covered.

I'll unsubscribe because I currently can't be part of the decision making process, but if someone wants to discuss a specific translation-y thing, it's okay to tag me, or poke me over at @maunzikation@cybre.space.

**Comment by gandaro** [comment]
* I do not like the current solution (“Profil”). As you already noticed, it’s impersonal, and does not really make sense. (A profile is a representation of some information stored in a database. A profile cannot “follow” you.)
* Cute language isn’t a problem. “Benutzi” is fine by me!
* We could also talk about “Wesen”. Some accounts might not be human (for instance, bot accounts).
* I have not heard about the “-lon” ending before. The only other word I know with a “-lon” ending is “Hässlon”, which is a nasty swear word.
* Do trans men really have a problem with *generisches Femininum* or is that just an assumption?
